src/models/FullDataset.ts

```diff
@@ -2,44 +2,65 @@
  * Common Indicator interface.
  * Do not use directly
  */
-export interface Indicator {
+export interface BaseIndicator {
+  short_name: string;
   value: number;
-  name: string;
-  description: string;
-  unit: "%" | "km / km^2" | "no / km^2";
+  description?: string;
```

Comment:
I am not 100% happy here, as `Indicator` have a `description`, but `KeyFigures` have a `definition`. Is this intended? 
That means `KeyFigures` will not receive a description, right?

Reply:
That means I would have to update the model to have the `description` only in `Indicator` and not in `KeyFigure`, where it is optional right now.

Reply:
Yes, kind of. I want to display a description of the indicator and what he is about. For the key figure, I would like to show the definition together with the data sources. The distinction of description and definition makes sense, here.

Reply:
Alright, I updated the model accordingly